### ISSUE TITLE
Next stable version when? any predictions?

### ISSUE THREAD

**Issue by oguzhankahyaoglu** [dotless/dotless#495]
Hi dude, again me :)

I am planning to upgrade our company's dotless&combres versions. I tested v1.5.2-beta4 and it seems stable for now, at least for my needs. We were using 1.3.1 ("strict math" is the only big difference as i see between these two versions, which i needed). You have been sending commits frequently. When do you plan to release a stable version? I can wait for your stable version too...

**Comment by rytmis** [comment]
1.5.2 should appear soon - it's basically a button click away. I'll take a look at your bug report first, though. 

1.5.x contains quite a few new features, including support for import options and better support for mixin calls that call generated selectors. Oh, and a nasty multithreading bug that gets corrected in 1.5.2...

**Comment by oguzhankahyaoglu** [comment]
okay then, my bug is not urgent for the moment. i would be gladful if you fix it though.
my problem was actually about negative values such as width:-99px was being rendered as width:- 99px; which is not understood by the chrome (v1.5.0). it is okay in beta4 and i can use this version without that color bugfix.

anyway, i wil wait for your release for a couple of weeks and then decide...

thanks for your efforts

Sent from my ASUS Zenfone2

-------- Original Message --------

**Comment by rytmis** [comment]
Check out http://www.nuget.org/packages/dotless/1.5.2

**Comment by oguzhankahyaoglu** [comment]
Perfect, i will give a try and inform you. probably, will work perfectly like beta4...

**Comment by rytmis** [comment]
Probably, yes, because it's exactly the same source. :)

**Comment by oguzhankahyaoglu** [comment]
Ops, you compiled it with 4.0 runtime:

Could not load file or assembly 'dotless.Core' or one of its dependencies. This assembly is built by a runtime newer than the currently loaded runtime and cannot be loaded

I was trying with your source code by only changing runtime version in beta4 version. If you are able to do it, you can support 3.5 framework by doing that, or you can also support both frameworks via nuget if it is possible (i am not sure about it).

**Comment by rytmis** [comment]
That's funny, the build runs on the CI server and I don't recall making any changes that should have affected the target runtime. Sigh... I'll look into it.

**Comment by rytmis** [comment]
Oh, right, now I remember. It's the ILMerge that does that. I've been trying to figure a way around it for a while now.

**Comment by oguzhankahyaoglu** [comment]
Okay, its not a very crucial thing for me, most of people moved to 4.0 framework, but we couldnt. I really wish to, but not now. As I said, I can solve my issue by using source code, but it would be delicious if u can manage that :+1:

**Comment by rytmis** [comment]
The obvious solution to this problem is not merging the assemblies (which paves way for splitting the NuGet packages as tracked by #449). Will do that at some point.

**Comment by oguzhankahyaoglu** [comment]
I saw that but i could not see the reason behing merging assemblies. I am using your source code via 2 assemblies. Are there any reason for merging them?

**Comment by rytmis** [comment]
The merge is a decision made before my time, and before NuGet was a thing -- I'm guessing ease of deployment was the goal. When I set up the CI server, I also upgraded some of the dependencies to NuGet packages instead of versioned binaries, which is when the newer ILMerge began to produce assemblies that target 4.0 instead of 3.5...

Currently, the reason I haven't dropped the ILMerge is that once I do that, I'll be forced to figure out the new package structure. Been putting that off for a while now. :)
